packages/webapp/src/containers/Animals/Inventory/useAnimalInventory.tsx

```diff
@@ -153,26 +153,32 @@ const formatAnimalsData = (
   defaultAnimalBreeds: DefaultAnimalBreed[],
   defaultAnimalTypes: DefaultAnimalType[],
 ) => {
-  return animals.map((animal: Animal) => {
-    return {
-      id: generateInventoryId(animal),
-      icon: getDefaultAnimalIcon(defaultAnimalTypes, animal.default_type_id),
-      identification: chooseIdentification(animal),
-      type: chooseAnimalTypeLabel(animal, defaultAnimalTypes, customAnimalTypes),
-      breed: chooseAnimalBreedLabel(animal, defaultAnimalBreeds, customAnimalBreeds),
-      groups: animal.group_ids.map((id: number) => getProperty(animalGroups, id, 'name')),
-      path: `/animal/${animal.internal_identifier}`,
-      count: 1,
-      batch: false,
-      // preserve some untransformed data for filtering
-      group_ids: animal.group_ids,
-      sex_id: animal.sex_id,
-      custom_type_id: animal.custom_type_id,
-      default_type_id: animal.default_type_id,
-      custom_breed_id: animal.custom_breed_id,
-      default_breed_id: animal.default_breed_id,
-    };
-  });
+  return animals
```

Comment:
Ultimately this filter would probably be put somewhere else (once we have a view for removed animals), but for now they are simply removed from the table + KPI